uber/site_sections/registration.py

```diff
@@ -55,13 +55,14 @@ def index(self, session, message='', page='0', search_text='', uploaded_id='', o
         attendees = attendees.order(order)
 
         groups = []
-        for group in session.query(Group) \
-                            .options(joinedload(Group.leader)) \
-                            .filter(Group.id.in_(
-                                session.query(Attendee.group_id)
-                                       .filter(Attendee.group_id != None, Attendee.first_name == '')
-                                       .distinct()
-                                       .subquery())).all():
+        for group in (session.query(Group)
+                             .options(joinedload(Group.leader))
+                             .filter(Group.status != c.WAITLISTED,
+                                     Group.id.in_(
+                                        session.query(Attendee.group_id)
+                                               .filter(Attendee.group_id != None, Attendee.first_name == '')
+                                               .distinct()
+                                               .subquery()))):
```

Comment:
This is mostly a style refactor, but I also added a clause to eliminate waitlisted dealers from the groups dropdown list.  We'll probably delete all of the waitlisted dealers before the event like we usually do, but just in case it seemed sensible to exclude them, since we don't want people able to check in by claiming those badges.